test/interop/fortran/FortranCallChapel/chapelProcs.skipif

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Is gfortran capable of working with C libraries created by compilers other than gcc? (icc might be fine but cce probably not, right?)  We might want to limit this skipif to cases where the compiler is gcc

Reply:
I've tested with gcc and clang and both worked.  The .skipif should already prevent it from running in places we test with icc or cce - they use launchers, and gfortran is only available with PrgEnv-gnu loaded.  I can add checks for icc and cce if you think it's still important.

Reply:
Nah, that seems reasonable.  Thanks!